### PR TITLE
Add cpu evict doc

### PR DESCRIPTION
**Pull request by zwzhang0107**

### CODE REVIEW

File: docs/user-manuals/cpu-evict.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# Eviction Strategy base on CPU Satisfaction
+
+## Introduction
+Koordinator supports [CPU Suppress](/docs/user-manuals/cpu-suppress) stategy, which is used for limiting the available
```
Comment (saintube) [inline docs/user-manuals/cpu-evict.md:4]: typo: `stategy`

File: docs/user-manuals/cpu-evict.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+   
+6. Run the following command through [stress tool](https://linux.die.net/man/1/stress)
+make sure the memory usage of node is above the threshold config, and the argument `--cpu`
+means the process will consume 10 cores, this should be adjusted according to the node capacity.
+
+   ```bash
+   $ stress --cpu 1 --vm 1 --vm-bytes 10G --vm-keep
+   ```
+   
+7. Check the running state of be-pod-demo, then you can find the be-pod-demo pod is not exsit,
```
Comment (saintube) [inline docs/user-manuals/cpu-evict.md:131]: typo: `exsit`

File: sidebars.js [hunk 3]
```diff
@@ -54,6 +54,7 @@ const sidebars = {
         'user-manuals/cpu-qos',
         'user-manuals/memory-qos',
         'user-manuals/memory-evict',
+        'user-manuals/cpu-evict',
```
Comment (saintube) [inline sidebars.js:57]: recommend placing after the CPU QoS

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Pull request merged**